Wrapper.php

```diff
@@ -361,12 +363,11 @@ public function stream_write($data)
      */
     public function dir_closedir()
     {
-        if (null === $handle = closedir($this->getStream())) {
-            $this->_stream     = null;
-            $this->_streamName = null;
-        }
+        closedir($this->getStream());
+        $this->_stream     = null;
```

Comment:
Maybe this test case can be removed then https://github.com/hoaproject/Protocol/blob/97a33357e9f0827b75e19a78429d037fab18c1ce/Test/Unit/Wrapper.php#L585-L601. Can you do that please?